conf/messages.cy

```diff
@@ -46,6 +46,7 @@ contact.extra-support.part3=' arnoch i lenwi ffurflenni, neu i gael fformat gwah
 reporting.heading=Rhoi gwybod am broblemau hygyrchedd gyda’r gwasanaeth hwn
 reporting.report-problem.part1='Rydym bob amser yn ceisio gwella hygyrchedd y gwasanaeth hwn. Os byddwch yn dod o hyd i unrhyw broblem nad yw wedi’i rhestru ar y dudalen hon, neu os ydych o’r farn nad ydym yn bodloni gofynion hygyrchedd, '
 reporting.report-problem.part2=rhowch wybod am y broblem hygyrchedd (yn agor ffenestr neu dab newydd)
+reporting.report-problem.part3=.
```

Comment:
Just a note on Play messages strings: if a message isn't defined in a non-default language file, it will fall back to the default (i.e., the same name in conf/messages). If we want to, we don't need to define a full stop in both message files, however it does no harm to have in both.

Reply:
Perfect, thanks.  I had meant to check that.